tasmota/tasmota_template.h

```diff
@@ -689,6 +692,14 @@ const uint8_t kGpioNiceList[] PROGMEM = {
 #ifdef USE_AS3935
   GPIO_AS3935,
 #endif
+#ifdef USE_TCP_BRIDGE
```

Comment:
Redundant. Should not be here. See several lines above.

Reply:
done